src/js/entities-service/entities/flow-actions.js

```diff
@@ -0,0 +1,66 @@
+import _ from 'underscore';
+import Radio from 'backbone.radio';
+import Store from 'backbone.store';
+import BaseCollection from 'js/base/collection';
+import BaseModel from 'js/base/model';
+
+const TYPE = 'flow-actions';
+
+const _Model = BaseModel.extend({
+  url() {
+    return `/api/flows/${ this.get('_flow') }/relationships/actions`;
+  },
+  type: TYPE,
+  getAction() {
```

Comment:
I believe this will be `_action` and no need for the `isNew` check we can assume there's always an action